extensions/binding/org.eclipse.smarthome.binding.tradfri/src/main/java/org/eclipse/smarthome/binding/tradfri/TradfriBindingConstants.java

```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.tradfri;
+
+import java.util.Set;
+
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+import com.google.common.collect.ImmutableSet;
+
+/**
+ * The {@link TradfriBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Kai Kreuzer - Initial contribution
+ */
+public class TradfriBindingConstants {
+
+    private static final String BINDING_ID = "tradfri";
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID GATEWAY_TYPE_UID = new ThingTypeUID(BINDING_ID, "gateway");
+
+    public final static ThingTypeUID THING_TYPE_DIMMABLE_LIGHT = new ThingTypeUID(BINDING_ID, "0100");
+    public final static ThingTypeUID THING_TYPE_COLOR_TEMP_LIGHT = new ThingTypeUID(BINDING_ID, "0220");
+    public final static ThingTypeUID THING_TYPE_DIMMER = new ThingTypeUID(BINDING_ID, "0820");
+
+    public static final Set<ThingTypeUID> SUPPORTED_LIGHT_TYPES_UIDS = ImmutableSet.of(THING_TYPE_DIMMABLE_LIGHT,
+            THING_TYPE_COLOR_TEMP_LIGHT);
+    public static final Set<ThingTypeUID> SUPPORTED_CONTROLLER_TYPES_UIDS = ImmutableSet.of(THING_TYPE_DIMMER);
+
+    // List of all Channel IDs
+    public static final String CHANNEL_BRIGHTNESS = "brightness";
+    public static final String CHANNEL_COLOR_TEMPERATURE = "colorTemperature";
```

Comment:
Nice. Thanks.

Pointing to [https://github.com/openhab/openhab2-addons/pull/1815#issuecomment-300994418](https://github.com/openhab/openhab2-addons/pull/1815#issuecomment-300994418) and [https://github.com/eclipse/smarthome/projects/2](https://github.com/eclipse/smarthome/projects/2) could this channel or these channels potential candidates for new system channels in the categories `ColorLight` and `DimmableLight`? Shall we open an issue for collection and discussion? Since I am already working on the translation of system channels, I am on topic and like to improve that part framework.